jobs/config.json

```diff
@@ -2973,16 +2973,37 @@
       "--gcp-project-type=gpu-project",
       "--gcp-zone=us-west1-b",
       "--provider=gce",
-      "--test_args=--kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8 --ginkgo.focus=\\[Feature:GPUUpgrade\\]",
+      "--test_args=--kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8 --ginkgo.skip=\\[.+\\]|Initializers|Dashboard",
```

Comment:
Why do we need to specify "Initializers|Dashboard"?

Reply:
cc @krousey 
I am following the example from ci-kubernetes-e2e-gke-staging-1-7-1-8-upgrade-master. Would also like to know what is the effect of skipping "Initializers|Dashboard". Do they cause us to skip the unnecessary e2e initialization and dashboard rows after upgrade test?

Reply:
Dashboard tests that the OSS kubernetes dashboard is available. GKE no longer has this as a default addon. IMHO it's a pretty questionable test and not particularly useful.